src/app/components/lunch/lunch.component.ts

```diff
@@ -126,5 +126,16 @@ export class LunchComponent implements OnInit {
 
 		return dates;
 	}
-
+	lunchClassMaker(classInput) {
```

Comment:
It might be best to have the `lunchClassMaker()` function take the input, convert everything to lowercase, and replace spaces with hyphens (`-`). This would eliminate these special hard-coded values and make it so adding another category simply requires a CSS rule